RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronCoreProducer.cc

```diff
@@ -1,62 +1,79 @@
 #include "DataFormats/EgammaCandidates/interface/GsfElectronCore.h"
-#include "DataFormats/EgammaCandidates/interface/GsfElectronCoreFwd.h"
```

Comment:
Still missing the point why these includes are removed, as well as ```SuperClusterFwd.h```

Reply:
Someone correct me if I'm wrong, but here is the line of thought I had:

In general, the point of forward declarations is to keep header files light. This `LowPtGsfElectronCoreProducer.cc ` is not a header file and therefore the actual `GsfElectronCore.h` is needed instead of `GsfElectronCoreFwd.h`. This makes `GsfElectronCoreFwd.h` superfluous because it's already included in `GsfElectronCore.h` [1], like it should be to get the typedefs.

[1] https://github.com/cms-sw/cmssw/blob/master/DataFormats/EgammaCandidates/interface/GsfElectronCoreFwd.h